CentralManagement/ChocoCCM Command Reference/Remove-CCMDeploymentStep.md

```diff
@@ -0,0 +1,106 @@
+---
+external help file: ChocoCCM-help.xml
+Module Name: ChocoCCM
+online version:
+schema: 2.0.0
+---
+
+# Remove-CCMDeploymentStep
+
+## SYNOPSIS
+Removes a deployment plan
+
+## SYNTAX
+
+```
+Remove-CCMDeploymentStep [-Deployment] <String> [-Step] <String> [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Removes the Deployment Plan selected from a Central Management installation

```

Comment:
This synopsis/description is duplicated from the `Remove-CcmDeployment` command.

Again with respect to endpoints, would be good if we can expose a set that works by ID as well. It's very easy to name multiple deployments or multiple steps the same thing, and it's not clear if that would just delete all matching ones, or throw an error and be impossible to do -- neither are great options.